BinaryEncoding.md

```diff
@@ -481,6 +481,12 @@ The function names subsection is a `name_map` which assigns names to
 a subset of the [function index space](Modules.md#function-index-space)
 (both imports and module-defined).
 
+Each function may be named at most once.
```

Comment:
I was thinking it would result in a malformed section.  Similar to other restrictions such as that functions must be named in index order.